datastore/cloud-client/admin.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2016, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START datastore_admin_client_create]
+from google.cloud.datastore_admin_v1.gapic import datastore_admin_client
+
+
+def client_create():
+    """Creates a new Datastore admin client."""
+    client = datastore_admin_client.DatastoreAdminClient()
+
+    print("Admin client created\n")
+    return client
+# [END datastore_admin_client_create]
+
+
+# [START datastore_admin_entities_export]
+def export_entities(project_id, output_url_prefix):
+    """
+    Exports a copy of all or a subset of entities from
+    Datastore to another storage system, such as Cloud Storage.
+    """
+    # project_id = "project-id"
+    # output_url_prefix = "gs://bucket-name"
+    client = datastore_admin_client.DatastoreAdminClient()
+
+    op = client.export_entities(project_id, output_url_prefix)
+    response = op.result()
```

Comment:
`op.result()` can wait forever. Can we have a timeout?

Reply:
No problem, pushed